eventing/README.md

```diff
@@ -50,22 +50,33 @@ systems using Kubernetes
 
 ## Installation
 
-You can install the core Knative Eventing (which provides an in-memory
-ChannelProvisioner) and the core sources (which provides the Kubernetes Events
-and "Container" Sources) with the following commands:
+Knative Eventing currently requires Knative Serving and Istio version 1.0 or
+later installed. Use this command to install the version of Istio which is
+tested with Knative:
+
+```shell
+kubectl apply --filename https://raw.githubusercontent.com/knative/serving/v0.2.2/third_party/istio-1.0.2/istio.yaml
+kubectl apply --filename https://github.com/knative/serving/releases/download/v0.2.2/release.yaml
+```
 
-<!-- TODO(evankanderson): Switch to a numbered release when available. -->
+You can install the core Knative Eventing (which provides an in-memory
+ChannelProvisioner) and the core sources (which provides the Kubernetes Events,
+GitHub, and "Container" Sources) with the following commands:
 
 ```bash
-kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing/latest/release.yaml
-kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing-sources/latest/release.yaml
+kubectl apply --filename https://github.com/knative/eventing/releases/download/v0.2.0/release.yaml
+kubectl apply --filename https://github.com/knative/eventing-sources/releases/download/v0.2.0/release.yaml
 ```
 
-In addition to the core sources, you can also use GCP PubSub as a source with
-the following yaml:
+In addition to the core sources, you can also use GCP PubSub as a source by
+creating a secret with the name `gcppubsub-source-key` with a `key.json` value
+and loading the released source yaml (the `-with-gcppubsub` release includes all
+the above sources, and adds GCP PubSub, which requires the listed secret):
 
 ```bash
-kubectl apply --filename https://storage.googleapis.com/knative-releases/eventing-sources/latest/release-with-gcppubsub.yaml
+kubectl create namespace knative-sources
+kubectl --namespace knative-sources create secret generic gcppubsub-source-key --from-literal=key.json=''
```

Comment:
seems like we should specify the key file here?

Reply:
Creating the key file requires a GCP account and a half-dozen `gcloud` commands. This command is sufficient to cause the controller to schedule, leaving the actual secret generation to the gcppubsub sample (see #559).

IMO, we should fix the gcppubsub source to load the secret dynamically per-Source rather than putting it in a Downward API volume.